internal/node/node.bzl

```diff
@@ -361,8 +361,8 @@ if (process.cwd() !== __dirname) {
         executable = ctx.outputs.launcher_sh
 
     # syntax sugar: allows you to avoid repeating the entry point in data
-    # entry point is only needed in runfiles if it is a .js file
-    if len(ctx.files.entry_point) == 1 and ctx.files.entry_point[0].extension == "js":
+    # entry point is only needed in runfiles if it is a javascript file
+    if len(ctx.files.entry_point) == 1 and ctx.files.entry_point[0].extension in ["js", "mjs"]:
```

Comment:
nit: maybe we could share a constant list with some other location like `is_js_file` helper or something? it feels always hard to know all the spots that need updates

Reply:
Updated